ogr/ogrfeature.cpp

```diff
@@ -562,6 +563,35 @@ OGRGeometryH OGR_F_StealGeometry( OGRFeatureH hFeat )
         OGRFeature::FromHandle(hFeat)->StealGeometry());
 }
 
+/************************************************************************/
+/*                       OGR_F_StealGeometryEx()                        */
+/************************************************************************/
+
+/**
+ * \brief Take away ownership of geometry.
+ *
+ * Fetch the geometry from this feature, and clear the reference to the
+ * geometry on the feature.  This is a mechanism for the application to
+ * take over ownership of the geometry from the feature without copying.
+ * This is the functional opposite of OGR_F_SetGeomFieldDirectly.
+ *
+ * After this call the OGRFeature will have a NULL geometry for the
+ * geometry field of index iGeomField.
+ * 
+ * @param hFeat feature from which to steal a geometry.
+ * @param iGeomField index of the geometry field to steal.
+ * @return the pointer to the stolen geometry.
```

Comment:
```suggestion
 * @return the pointer to the stolen geometry.
 * @since GDAL 3.5
```

Reply:
Done in commit [1cf7769](https://github.com/OSGeo/gdal/pull/5196/commits/1cf7769cf09c20b492836dc209ff16898ab404ae)